analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -48,9 +48,11 @@
 
 import awkward as ak
 import cabinetry
+import correctionlib
 from coffea import processor
 from coffea.nanoevents import NanoAODSchema
-
+from coffea.analysis_tools import PackedSelection
+from servicex import ServiceXDataset
```

Comment:
```suggestion
```

We don't need that import here anymore, it moved to `utils`. This probably came in while updating against the changes in `main`.